demisto_sdk/commands/create_id_set/create_id_set.py

```diff
@@ -70,6 +70,11 @@ def create_command_to_implemented_integration_map(self):
                     command_name_to_implemented_integration_map[command] = [integration_name]
         return command_name_to_implemented_integration_map
 
+    def default_json_serialize(self, obj):
+        if isinstance(obj, set):
+            return list(obj)
+        raise TypeError
```

Comment:
not sure what is the point of this?

Reply:
for converting sets to list before json serialising. Handled that inside the parsing func eventually and didn't delete that by mistake